generator.rb

```diff
@@ -0,0 +1,111 @@
+def is_vowel(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    return "true"
+  else
+    return "false"
+  end
+end
+
+def rev_str(strg)
+  aa = strg.split("")
+  rev = Array.new
+  aa.length.times do
```

Comment:
Could you use more specific variable names here?
- `rev_str` could be `reverse_string`
- `aa` could be `string_array`
- `rev` could be `reversed`

IMO, that would make this code much easier to read.

Reply:
Also, Ruby has a way to do this built in: `String#reverse`

```ruby
puts "abcd".reverse
# => dcba
```